integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KafkaClusterConfig.java

```diff
@@ -138,6 +136,23 @@ public Stream<ConfigHolder> getBrokerConfigs(Supplier<KafkaEndpoints> endPointCo
                 server.put(String.format("listener.name.%s.plain.sasl.jaas.config", "EXTERNAL".toLowerCase()), plainModuleConfig);
             }
 
+            if (securityProtocol != null && securityProtocol.contains("SSL")) {
```

Comment:
Will the protocol always be `SSL` or should we cater for `TLS` as well?

Reply:
The security protocol enum has two different values for TLS: SSL and SASL_SSL.
* SASL_SSL uses TLS to encrypt communication but SASL to authenticate the user
* SSL uses TLS for encryption and authentication